infra/modules/providers/azure/container-registry/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "resource_group_name" {
+  description = "(Required) The name of the resource group in which to create the Container Registry. Changing this forces a new resource to be created."
+  type        = "string"
+}
+
+variable "acr_name" {
+  description = "(Required) Specifies the name of the Container Registry. Changing this forces a new resource to be created."
+  type        = "string"
+}
+
+variable "acr_location" {
```

Comment:
this variable appears to be unused in the template. I agree with your approach of defaulting the location to the RG. We could probably remove this.

Reply:
I noticed this and removed it in a push earlier today, so it's already taken care of